comet-java-client/src/test/java/ml/comet/experiment/impl/OnlineExperimentTest.java

```diff
@@ -445,22 +448,66 @@ public void testUploadAndGetAssets() {
     }
 
     @Test
-    public void testSetsContext() {
-        OnlineExperiment experiment = createOnlineExperiment();
+    public void testLogAndGetRemoteAssets() {
+        try (OnlineExperimentImpl experiment = (OnlineExperimentImpl) createOnlineExperiment()) {
+            // Make sure experiment has no assets
+            //
+            assertTrue(experiment.getAssetList(ASSET_TYPE_ALL).isEmpty());
+
+            // Log remote assets and wait for completion
+            //
+            ExperimentContext context = new ExperimentContext(10, 101, "train");
+            OnCompleteAction onComplete = new OnCompleteAction();
+            Map<String, Object> metadata = new HashMap<String, Object>() {{
```

Comment:
check if you use metadata in other tests, if so move to const, share in all tests

Reply:
Only single place as for now.